addons/mail/static/src/discuss/call/public_web/discuss_sidebar_call_participants.js

```diff
@@ -23,7 +26,19 @@ export class DiscussSidebarCallParticipants extends Component {
             onHover: () => (this.floating.isOpen = true),
             onAway: () => (this.floating.isOpen = false),
         });
+        this.state = useState({ expanded: false });
         this.floating = useDropdownState();
+        useEffect(
+            (selfSession, compact) => {
+                if (selfSession?.in(this.sessions)) {
+                    this.state.expanded = true;
+                }
+                if (compact) {
+                    this.state.expanded = false;
+                }
```

Comment:
Not a huge deal, but changing synchronously value twice can lead to unnecessary burden for inner-code of reactive: the notifyReactive happens synchronously, so the 1st `expanded = true` will notify. Most observers are components so they are batched in micro-task tick at most, so they won't process until potential `expanded = false` that would tell not to render... But you could avoid accessing and notifying observers for nothing by not setting a change of value if you know you don't have to change the value :)